src/python/CMakeLists.txt

```diff
@@ -104,15 +118,14 @@ function(PYIMATH_EXTRACT_REL_SITEARCH varname pyver pyexe pysitearch)
 endfunction()
 
 if(Python2_FOUND)
-  set(PYIMATH_BOOST_PY2_COMPONENT "python${Python2_VERSION_MAJOR}${Python2_VERSION_MINOR}")
+  set(PYIMATH_BOOST_PY_COMPONENT "python${Python2_VERSION_MAJOR}${Python2_VERSION_MINOR}")
   message(STATUS "Found Python2 libraries: ${Python2_VERSION_MAJOR}${Python2_VERSION_MINOR}")
   # we can't just use the Python2_SITEARCH variable as that then will
   # ignore CMAKE_INSTALL_PREFIX. Could extract this to a function somewhere
   # if it is generally useful
   pyimath_extract_rel_sitearch(PyImath_Python2_SITEARCH_REL 2 ${Python2_EXECUTABLE} ${Python2_SITEARCH})
-endif()
-if(Python3_FOUND)
-  set(PYIMATH_BOOST_PY3_COMPONENT "python${Python3_VERSION_MAJOR}${Python3_VERSION_MINOR}")
+elseif(Python3_FOUND)
+  set(PYIMATH_BOOST_PY_COMPONENT "python${Python3_VERSION_MAJOR}${Python3_VERSION_MINOR}")
   message(STATUS "Found Python3 libraries: ${Python3_VERSION_MAJOR}${Python3_VERSION_MINOR}")
   # and figure out the install root here
   pyimath_extract_rel_sitearch(PyImath_Python3_SITEARCH_REL 3 ${Python3_EXECUTABLE} ${Python3_SITEARCH})
```

Comment:
Just noticed, that these blocks can probably be combined as well.
If we set Python_EXECUTABLE and Python_SITEARCH above after line 56 and 70 resp., we might be able to just use one block with the 3 statements and without the guards.
What do you think?

Reply:
Tested it against python 3 and it works. Gonna push it with the old code commented, so you can check it.